plugins/meta/flannel/flannel_windows.go

```diff
@@ -65,10 +65,13 @@ func doCmdDel(args *skel.CmdArgs, n *NetConf) error {
 		return err
 	}
 
+	// cleanup will work when no error happens
+	defer cleanup(err)
```

Comment:
I think this doesn't work because it gets the value of `err` at the time control passes through this line.
See https://play.golang.org/p/3W5_yxRJJ_6 vs https://play.golang.org/p/-9c9Z3pvVFO

Reply:
The alternative is to make your cleanup function take a *pointer* to the error, and then `defer cleanup(&err)` and then it'll work :)

Reply:
updated, thanks!